src/gradle.ts

```diff
@@ -84,8 +84,13 @@ export function getTaskToPublish(
             reject(new Error(ERROR_MULTIPLE_PLUGIN));
           }
           task = "publishPlugins";
+        } else {
+          logger.debug(line.toString())
         }
```

Comment:
I think `else` is needless, better to print all tasks for debugging purpose. Would you mind if we change our code like below?

```suggestion
        }
        logger.debug(line.toString());
```